src/data/cassandra/SettingsProvider.cpp

```diff
@@ -36,43 +36,26 @@
 #include <ios>
 #include <iterator>
 #include <optional>
-#include <stdexcept>
 #include <string>
+#include <string_view>
 #include <system_error>
 
 namespace data::cassandra {
 
 namespace impl {
-inline Settings::ContactPoints
-tag_invoke(boost::json::value_to_tag<Settings::ContactPoints>, boost::json::value const& value)
-{
-    if (not value.is_object()) {
-        throw std::runtime_error("Feed entire Cassandra section to parse Settings::ContactPoints instead");
-    }
-
-    util::Config const obj{value};
-    Settings::ContactPoints out;
-
-    out.contactPoints = obj.valueOrThrow<std::string>("contact_points", "`contact_points` must be a string");
-    out.port = obj.maybeValue<uint16_t>("port");
-
-    return out;
-}
 
 inline Settings::SecureConnectionBundle
-tag_invoke(boost::json::value_to_tag<Settings::SecureConnectionBundle>, boost::json::value const& value)
+invoke_tag_SecureConnections(std::string_view value)
```

Comment:
In any case the function name should change if it had to stay. The reason we had `tag_invoke` is because it's one of the "standard" approaches to provide customization points. We should avoid using snake_case other than for `tag_invoke` and maybe `make_*` (should probably call them `make*` in the first place).